assets/pages/regular-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -9,23 +9,16 @@ import ContributionsCheckout from 'components/contributionsCheckout/contribution
 
 import { type PageState as State } from '../regularContributionsReducer';
 import RegularContributionsPayment from './regularContributionsPayment';
-import RegularInlineContributionsPayment from './regularInlineContributionsPayment';
 
 
 // ----- State Maps ----- //
 
 function mapStateToProps(state: State) {
-
-  const inlineCardPaymentVariant = state.common.abParticipations.inlineStripeFlowCardPayment;
-
   return {
     amount: state.page.regularContrib.amount,
     currencyId: state.common.internationalisation.currencyId,
     country: state.common.internationalisation.countryId,
-    inlineCardPaymentVariant,
-    payment: inlineCardPaymentVariant === 'inline' ?
-      <RegularInlineContributionsPayment /> :
-      <RegularContributionsPayment />,
+    payment: <RegularContributionsPayment />,
```

Comment:
Same comment as above, could be passed through in `regularContributions.jsx`.

```jsx
<ContributionsCheckoutContainer
  contributionType={contributionType}
  form={<FormFields />}
  payment={<RegularContributionsPayment />}
/>
```

Reply:
Fixed here: a163628 good spot!